src/sass/modules/_m-schemaform.scss

```diff
@@ -14,9 +14,6 @@ fieldset {
   display: inline-block;
   width: 100%;
   max-width: 750px;
-  @media (min-width: $large-screen) {
```

Comment:
@cehsu I couldn't find anything that I broke by removing this, but do you remember what it was added for?

Reply:
I'm not sure which specific screen size/browser/view scenario this addresses. The comment indicates this is to address an IE bug. It is part of a bug fix [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/7338) for the photo tool page and its view within the review page. I'll test this out and report back.

Reply:
I tested on IE and didn't notice any issues on the review page, at least.

Reply:
The issue is on the normal cropper page, not the review page:
<img width="774" alt="screen shot 2018-03-09 at 3 03 03 pm" src="https://user-images.githubusercontent.com/16051172/37234201-47f9ca54-23ab-11e8-9fee-2f742592110c.png">
An issue has been created for this bug [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9197).